### ISSUE TITLE
ModuleNotFoundError: No module named 'vine.five'

### ISSUE THREAD

**Issue by jokiefer** [mrmap-community/mrmap#23]
# Status quo
On a fresh installed debian 10 we have struggle with the current celery package `celery==4.3.0`

Running `runserver` command runs in to python-BaseException:
```
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/jonas/Git/MrMap/MrMap/__init__.py", line 5, in <module>
    from .celery_app import app as celery_app
  File "/home/jonas/Git/MrMap/MrMap/celery_app.py", line 12, in <module>
    from celery import Celery, task
  File "/home/jonas/Git/MrMap/venv/lib/python3.7/site-packages/celery/__init__.py", line 153, in <module>
    from . import local  # noqa
  File "/home/jonas/Git/MrMap/venv/lib/python3.7/site-packages/celery/local.py", line 17, in <module>
    from .five import PY3, bytes_if_py2, items, string, string_t
  File "/home/jonas/Git/MrMap/venv/lib/python3.7/site-packages/celery/five.py", line 7, in <module>
    import vine.five
ModuleNotFoundError: No module named 'vine.five'
python-BaseException

```

**Comment by jokiefer** [comment]
# workaround
downgrade vine-5.0.0 to 1.3.0 solves the problem:
```
pip uninstall vine
pip install vine==1.3.0
```

**Comment by jokiefer** [comment]
@joki I think we encountered the same problem in https://git.osgeo.org/gitea/GDI-RP/MrMap/pulls/483

This could be fixed by upgrading `celery` to version `4.3.1`. So a downgrade of `vine` should not be necesssary

**Comment by jokiefer** [comment]
since #64 is fixed, celery is now on version 5.0.5.
This error can't reproduced at the current master. So i think we can close this.
